### PR TITLE
Add performance audit

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR, I:

✔️ Added the email column to the owners' table
✔️ Added data to the visits' and to the owners' tables
✔️ Ran the "explain analyse" query on the visits counts for the animal with id=4, Execution time: 1133.868 ms (see screenshot 1); the vet with id = 2, Execution Time: 1129.756 ms (see screenshot 2) and for the owner with email = 'owner_18327@mail.com',  Execution Time: 2775.187 ms (see screenshot 3).
image

image

image

✔️ Found a way to decrease the execution time of the first query based on the hints given in the materials.
✔️ Found a way to improve the execution time of the other two queries.
✔️ While making changes, I checked if they helped by running EXPLAIN ANALYZE. Once I succeeded, took a screenshot of the EXPLAIN ANALYZE result showing that time actually decreased.
✔️ Added any changes to the database schema to the schema.sql file and committed in a new pull request.
✔️ Included screenshots with EXPLAIN ANALYZE results in my PR description (before and after I improved the speed) for each of the 3 problematic queries.

image

image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -289,4 +289,83 @@ ON a.specy_id = s.id
...
+FROM owners
+GROUP BY full_name, age, email
+HAVING COUNT(*) > 1;
+CREATE INDEX duplicates_to_keep_index ON duplicates_owners_to_keep (keep_id);
+COMMIT;
+  
+/* query to delete duplicate owners not applied */
+DELETE FROM visits
+WHERE id NOT IN (SELECT keep_id FROM duplicates_owners_to_keep);
+COMMIT;
```
Comment (omar-muhamad) [inline queries.sql]: - [x] [Optional] [Highly recommended] Kindly delete all this code because it's not useful in this milestone.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi team,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (diarisdiakite) [reply queries.sql]: Well received @Omar-Muhamad 
As we used it to improve performance, we thought it was important to show how we did. 
Thank you for the comment.

**Comment by diarisdiakite** [comment]
In this Milestone, I added the schema capture.

image

**Pull request merged**